apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -120,8 +121,12 @@ const REPOSITORIES = [JobRepository];
 
 const SERVICES: Provider[] = [
   {
-    provide: MetricQueueService,
-    useClass: MetricQueueService,
+    provide: MetricQueueCompletedService,
+    useClass: MetricQueueCompletedService,
+  },
+  {
+    provide: MetricQueueActiveService,
+    useClass: MetricQueueActiveService,
```

Comment:
@LetItRock Is the below code on line 144 still the best way to get all of the queues?

```
 provide: 'BULLMQ_LIST',
    useFactory: (workflowQueue: QueueService, triggerQueue: TriggerQueueService, wsQueue: WsQueueService) => {
      return [workflowQueue, triggerQueue, wsQueue];
},
```